applications/conceptual-model-editor/src/app/diagram/reactflow/components/menu-options.tsx

```diff
@@ -43,40 +45,74 @@ export const useMenuOptions = () => {
         deleteHandler?: () => void;
         position?: string;
     }) => {
-        const { openDetailHandler, createProfileHandler, modifyHandler, removeFromViewHandler, deleteHandler } = props;
+        const { openDetailHandler, createProfileHandler, modifyHandler, removeFromViewHandler, deleteHandler, position } = props;
+        // You have to actually define the type here explictly, typescript can't interfere the type correctly
+        const handlersWithTexts: Array<[(() => void) | undefined, string]> = [      
+            [openDetailHandler, "ℹ Detail"],
+            [modifyHandler, "✏ Modify"],
+            [undefined, "HorizontalSeparator"],
+            [createProfileHandler, "🧲 Create profile"],            
+            [undefined, "HorizontalSeparator"],
+            [removeFromViewHandler, "🕶 Remove from view"],
+            [deleteHandler, "🗑 Delete"],
+        ];
+        return MenuOptionsGeneral({handlersWithTexts, positionTailwind: position});        
+    };    
+
+    /**
+     * 
+     * @param props is an object with following properties:
```

Comment:
Is this a valid comment block for the param? I would expect there need to be a leading spaces or two.